base/libc.jl

```diff
@@ -96,114 +98,38 @@ else
     error("systemsleep undefined for this OS")
 end
 
-immutable TimeVal
-   sec::Int64
-   usec::Int64
-end
-
-function TimeVal()
-    tv = Ref{TimeVal}()
-    status = ccall(:jl_gettimeofday, Cint, (Ref{TimeVal},), tv)
-    status != 0 && error("unable to determine current time: ", status)
-    return tv[]
-end
-
 """
-    TmStruct([seconds])
```

Comment:
And I don't think we can just remove things like this. It's also better to start with smaller changes one at a time.